examples/tty_conv.c

```diff
@@ -6,7 +6,11 @@
 #include <string.h>
 #include <errno.h>
 #include <unistd.h>
+#ifdef __GLIBC__
 #include <termios.h>
+#else
+#include <asm-generic/termios.h>
+#endif
```

Comment:
Could you elaborate what is the build issue you are fixing, please?

By the way, it seems to me that it should rather be
```
#include <asm/termios.h>
```
unconditionally.

Reply:
I assume musl libc has no termios.h header.

In any case, the logic is wrong. termios.h is not glibc specific but a standard. If an implementation does not support it, we need a workaround for this specific implementation and not break everything else.